src/components/MultiDeviceCodeEditor/index.tsx

```diff
@@ -14,10 +14,17 @@ import {
 	ResponsivePreviewTrigger,
 } from "./ResponsivePreview";
 
+const SHOW_CSS_FILES: "initial" | "final" = "initial";
+
 export function MultiDeviceCodeEditor(props: SandpackProviderProps) {
 	const firstFile = Object.keys(props?.files || {})[0];
 	const responsivePreviewId = useId();
 
+	if (SHOW_CSS_FILES === "final" && props.files?.["/responsive-final.css"]) {
+		props.files["/responsive.css"] = props.files["/responsive-final.css"];
+	}
+	delete props.files?.["/responsive-final.css"];
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏First time I see this operator 😮

Reply:
En général faut éviter le delete , en plus là je modifie les props directement, c'est clairement pas dingue 😅